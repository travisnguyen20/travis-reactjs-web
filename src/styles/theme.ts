import { DefaultTheme } from 'styled-components/macro';

const theme: DefaultTheme = {
  colors: {
    primaryBlue: '#000852',
    white: '#ffffff',
  },
  fontSizes: {
    small: '0.86rem',
    body: '1rem',
    h6: '1.14rem',
    h5: '1.43rem',
    h4: '1.71rem',
    h3: '2rem',
    h2: '2.29rem',
    h1: '3rem',
  },
  lineHeights: {
    h1: '3.42rem',
    h2: '2.86rem',
    h3: '2.57rem',
    h4: '2.29rem',
    h5: '2rem',
    h6: '1.71rem',
    small: '1.29rem',
    body: '1.43rem',
  },
  fontWeights: {
    regular: 400,
    medium: 500,
    semiBold: 600,
    bold: 700,
  },
};

export default theme;
