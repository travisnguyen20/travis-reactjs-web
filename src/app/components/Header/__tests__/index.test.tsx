import * as React from 'react';
import { render } from '@testing-library/react';

import { Header } from '..';
import { ThemeProvider } from 'styled-components';
import theme from 'styles/theme';

describe('<Header  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <ThemeProvider theme={theme}>
        <Header />
      </ThemeProvider>,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
