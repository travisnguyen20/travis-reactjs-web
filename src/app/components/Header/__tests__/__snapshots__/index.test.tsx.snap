// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header  /> should match snapshot 1`] = `
.c0 {
  top: 0;
  right: 0;
  left: 300px;
  height: 80px;
  background: #ffffff;
  box-shadow: 1px 1px 10px 2px rgb(0 0 0 / 20%);
}

.c1 {
  padding: 10px;
  height: 40px;
  width: 40px;
  cursor: pointer;
}

.c1 img {
  height: 100%;
  width: 100%;
}

<div
  class="c0 position-fixed d-flex align-items-center justify-content-end p-3"
>
  <div
    class="c1"
  >
    <img
      alt="help"
      src="information.svg"
    />
  </div>
  <div
    class="c1"
  >
    <img
      alt="notification"
      src="bell.svg"
    />
  </div>
  <div
    class="c1"
  >
    <img
      alt="chat"
      src="chat.svg"
    />
  </div>
  <div
    class="c1"
  >
    <img
      alt="username"
      src="user.svg"
    />
  </div>
</div>
`;
